src/Analyzers/Core/CodeFixes/FileHeaders/AbstractFileHeaderCodeFixProvider.cs

```diff
@@ -153,8 +158,12 @@ private SyntaxNode ReplaceHeader(Document document, SyntaxNode root, string expe
                 triviaList = triviaList.RemoveAt(removalList[i]);
             }
 
+#if CODE_STYLE
+            var newLineText = Environment.NewLine;
+#else
             var newLineText = document.Project.Solution.Options.GetOption(FormattingOptions.NewLine, root.Language);
-            var newLineTrivia = SyntaxGenerator.GetGenerator(document).EndOfLine(newLineText);
+#endif
```

Comment:
📝 We can file a follow-up for this if it matters. One way or another the situation should be easier to deal with in the future.

Reply:
Why did you need to change this code? I believe `FormattingOptions.NewLine` should be available from CodeStyle fixes layer and I am presuming it will always return the default value in CodeStyle layer (at least till it is switched into an editorconfig location based option), which should be `Environment.NewLine`. Basically, I am claiming that reverting this change will not break any functionality or tests.

Reply:
The type exists with the same name in two assemblies. It's resolvable via `extern alias`, except in this case it's the _workspaces_ layer option that we want to use.

Reply:
`FormattingOptions` is defined in a separate namespace in CodeStyle layer, so you should not require an external alias

Reply:
Oh, it seems we did not move `FormattingOptions` to a different namespace. We should do so to avoid extern aliases, and once all your Options infrastructure changes go in that can be reverted. This will match the approach taken for all other Options related types and avoid extern aliases.

Reply:
I can do the change I suggested here as a separate follow-up PR.